provisioner/zalando_eks.go

```diff
@@ -0,0 +1,178 @@
+package provisioner
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando-incubator/cluster-lifecycle-manager/api"
+	"github.com/zalando-incubator/cluster-lifecycle-manager/channel"
+	"github.com/zalando-incubator/cluster-lifecycle-manager/pkg/aws/eks"
+	"github.com/zalando-incubator/cluster-lifecycle-manager/pkg/decrypter"
+	"github.com/zalando-incubator/cluster-lifecycle-manager/pkg/util/command"
+)
+
+type (
+	ZalandoEKSProvisioner struct {
+		clusterpyProvisioner
+	}
+
+	ZalandoEKSModifier struct{}
+)
+
+// NewZalandoEKSProvisioner returns a new provisioner capable of provisioning
+// EKS clusters by passing its location and and IAM role to use.
+func NewZalandoEKSProvisioner(
+	execManager *command.ExecManager,
+	secretDecrypter decrypter.Decrypter,
+	assumedRole string,
+	awsConfig *aws.Config,
+	options *Options,
+) Provisioner {
+	provisioner := &ZalandoEKSProvisioner{
+		clusterpyProvisioner: clusterpyProvisioner{
+			provider:          ZalandoEKSProvider,
+			awsConfig:         awsConfig,
+			assumedRole:       assumedRole,
+			execManager:       execManager,
+			secretDecrypter:   secretDecrypter,
+			manageMasterNodes: false,
```

Comment:
Is `manageMasterNodes` actually used anywhere? I can't find it in this PR and since it is first introduced here I would expect it here somewhere.

Reply:
:facepalm: forgot to uncomment code, update here https://github.com/zalando-incubator/cluster-lifecycle-manager/commit/cf74c3c29154fb74f67ef8079f95454b1035b50a